components/bl-pdfViewer-component/src/index.js

```diff
@@ -47,8 +50,10 @@ export default function PdfViewer({ component, eventHandlers }) {
   };
 
   const onPageLoadSuccess = () => {
+    const spaceForControls = getBottomOffset(controlsRef.current) - getBottomOffset(pageRef);
+
+    pageRef.firstChild.style.height = `calc(${ height } - ${ spaceForControls }px)`;
```

Comment:
Why do we need calc here?

Reply:
need to specify the dimensions for the element that displays the pdf, if we use height that comes from the property, the block with control elements will fall out of the root element. Therefore, from the height that comes from the properties, I subtract the necessary space for displaying the control elements and assign it to the element that displays the pdf

Reply:
I subtracted it through calc because `height` comes with a unit of measurement (px, %, etc.), and `spaceForControls` is just a number